js/custom_script.js

```diff
@@ -256,6 +256,24 @@ $(function () {
     });
 });
 
+axios.post('/.netlify/functions/fetch_blogs').then((result) => {
+    let blogUl = document.getElementById('blog-ul');
+    const articles = result.data.data.user.publication.posts;
```

Comment:
is there any way to fetch only 2 blog posts?

Reply:
not possible, by default gets 6 data